components/Tabs/tests/snapshot/TabBarView/MDCTabBarViewLayoutStyleSnapshotTests.m

```diff
@@ -284,6 +308,30 @@ - (void)testFixedClusteredCenteredLayoutStyleTooWideSizeArabicRTL {
   [self generateSnapshotAndVerifyForView:self.tabBarView];
 }
 
+- (void)testFixedClusteredCenteredLayoutContentPaddingLatinLTR {
+  // When
+  self.tabBarView.preferredLayoutStyle = MDCTabBarViewLayoutStyleFixedClusteredCentered;
+  [self.tabBarView setContentPadding:UIEdgeInsetsMake(10, 20, 40, 30)
+                      forLayoutStyle:MDCTabBarViewLayoutStyleFixedClusteredCentered];
+
+  // Then
+  [self sizeViewToIntrinsicContentSize:self.tabBarView extraSize:CGSizeMake(100, 0)];
```

Comment:
In my opinion, it's not clear why you are adding extra size here. Same for the tests below.

Reply:
Generally, it's just to make sure there's sufficient `bounds` so that the Tab Bar doesn't "fall back" to another layout style, like `.Scrollable`.

Specifically for the Clustered layout styles, it helps to verify that it is being laid-out in a clustered manner and not filling the full bounds with the subviews. 

Do you think I need to add comments explaining why extra space was added in each case?

Reply:
Yeah, I'd add a comment or name the value as some variable with a meaningful contextual name. That way it's easier to recognize what the intention is.

Reply:
Added comments. Thanks!